src/main/resources/application.properties

```diff
@@ -1,16 +1,17 @@
 spring.application.name=eiffel-vici
 server.port=8080
-
 # REST api returns json in pretty format with jackson
 #spring.jackson.serialization.INDENT_OUTPUT=true
 spring.jackson.default-property-inclusion=non_null
-
 # LOGGING
 # Enables debug log
 #debug=true
 # Logging file, will not put out to a file if not defined.
 #logging.file=vici.log
-
 # Logging
 logging.level.root=INFO
-logging.level.org.springframework.web=INFO
\ No newline at end of file
+logging.level.org.springframework.web=INFO
+# Settings
+eiffel.vici.settings-file-name=settings.json
```

Comment:
What is the content of settings.json? Can't allsettings be located in this application.properties file?

Reply:
The idea was to let a single instance of Eiffel Vici be able to handle multiple sources (CI-systems?) which may want to have specific settings depending on how the system is build and what data is available.
Maybe this is not needed anymore in that case we should create a new issue about that

Reply:
ok